### DISCUSSION TITLE
Ghost database name when trying to use docker for testing

### DISCUSSION THREAD

**Discussion by dblack** [prisma/prisma#10651] [Q&A] [answered]
Hello --

I'm following this post: https://dev.to/eddeee888/how-to-write-tests-for-prisma-with-docker-and-jest-593i, and finding that whatever I do -- changing my original schema file, re-generating prisma, etc. -- I get this output when I try to create a migration via docker: 
```
Datasource "db": MySQL database "wishes_next" at "localhost:3306"
```
where `wishes_next` is the name of my real (not test) database but does not occur as a string anywhere in my repo. Things then fail, though probably for other reasons. 

I'm wondering where it's getting the name `wishes_next` from, since I've (for testing/learning purposes) expunged it. I'm hoping there's a prisma-related answer but if it turns out to be docker-specific I'll move on :-) Thanks.

**Comment by dblack** [comment, marked as answer]
Update: I think it is indeed a docker thing. I haven't gotten the whole blog post instructions working yet but I think the database name was probably a matter of caching in docker.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
